### DISCUSSION TITLE
Trading bot

### DISCUSSION THREAD

**Discussion by YousifAbdulla** [community/community#114602] [Programming Help]
### Body

I need a trading bot with python code language that will do the following: 
Trades 24/7
Hedged
It has footprint tool ( accurate signals)
Connected to the news
Pauses during the news
Always aims for higher profits
Automatic lot size
Managing account capital
Always updated 
Make sure that it has all these features

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by queenofcorgis** [comment]
Hi there, 

Your post was closed and locked because it violated this category's [guidelines](https://github.com/orgs/community/discussions/51384), specifically:

> ❌What is not allowed in this category:
> 
> * Asking people to code for you 

If you continue to post content that is not allowed,  you may receive a temporary ban 🛑.
